app.js

```diff
@@ -64,20 +62,42 @@ var appName = "thimble",
     ], {
       autoescape: true
     }),
-    parameters = require('./lib/parameters')(env),
-    routes = require('./routes')( utils, env, nunjucksEnv, appName ),
-    webmakerAuth = new WebmakerAuth({
-      forceSSL: env.get('FORCE_SSL'),
-      loginHost: env.get('APP_HOSTNAME'),
-      loginURL: env.get('LOGIN_URL'),
-      authLoginURL: env.get('LOGIN_URL_WITH_AUTH'),
-      secretKey: env.get('SESSION_SECRET'),
-      domain: env.get('COOKIE_DOMAIN')
-    }),
-    webmakerProxy = require('./lib/proxy')(env),
+    parameters = require('./lib/parameters')(),
+    routes = require('./routes')( utils, nunjucksEnv, appName ),
+    webmakerProxy = require('./lib/proxy')(),
 
     bracketsPath;
 
+function configuredCookieSession() {
```

Comment:
`cookieSession()` returns the cookieSession middleware, so I'm following that convention with `configuredCookieSession()` returning configured cookieSession middleware.